medallion/views/__init__.py

```diff
@@ -2,3 +2,4 @@
 MEDIA_TYPE_TAXII_V20 = "{media}; version=2.0".format(media=MEDIA_TYPE_TAXII_ANY)
 MEDIA_TYPE_STIX_ANY = "application/vnd.oasis.stix+json"
 MEDIA_TYPE_STIX_V20 = "{media}; version=2.0".format(media=MEDIA_TYPE_STIX_ANY)
+FILTERS = ["id", "type", "version", "added_after", "match"]
```

Comment:
`FILTERS` should probably be a tuple rather than a list, but this is really minor.

Reply:
I don't think there are any negative impacts to permitting all of these as the `allowed` argument to a filter, but if this is really a constant throughout the application, would it make more sense to just use it where needed rather than passing it through everywhere? I'll admit I haven't traced everywhere this is coming from, so I could be missing something.

Reply:
Previously the allowed filters was a hard coded tuple in the objects and manifests views that didn't include the added_after and match filters. I assume this was because these two filters previously didn't perform well once you got past a non-trivial number of indicators. I could have just added the two other filters to those tuples, but wanted one place to manage them. Happy to go with whatever option you think is best.
Will the concept of "allowed filters" be removed at some point? Re-reading the TAXII spec now I assume all the filters are required (although the spec isn't 100% explicit that this is the case).

Reply:
Yes, it's possible we could remove "allowed filters" at some point. My question was really addressed to @rpiazza, sorry.